docs/posts/roadmap-2024-H1/index.qmd

```diff
@@ -0,0 +1,381 @@
+ecosystem. Voltron Data supports Ibis because it is the Python frontend for
+Theseus and can act as a standard Python dataframe API for **any** backend,
+whether you're querying a CSV file on your laptop, running thousand-node Spark
+jobs on a cluster in the cloud, or doing cutting-edge work on-premise with GPU
+clusters. With Ibis, you can write your experimentation code for your laptop and
+scale up to Theseus (or any other backend) seamlessly.
+
+::: {.callout-note collapse="true" title="Why not the pandas API?"}
+This is a great, and natural, question -- if Voltron Data wants a standard
+Python dataframe API, why not just use pandas? The reason is relatively simple:
+the pandas API inherently does not scale. This is largely due to the expectation
+of ordered results and the index. pandas is implemented for single-threaded
+execution and has a lot of baggage when it comes to distributed execution. While
+projects like Modin and pandas on Spark (formerly Koalas) attempt to scale the
+pandas API, any project that attempts the feat is doomed to a dubious support
+matrix of operations.
+
+Instead, Wes McKinney envisioned Ibis as a portable Python dataframe where the
+API is decoupled from the execution engine. Ibis code scales to the backend it
+is connected to. Any other Python dataframe library locks you into its execution
+engine. While they may claim to be easy to migrate to, this is rarely the case.
+The founders of Voltron Data experienced these pains with the pandas API
+themselves in previous efforts, including cuDF. For Theseus and as an
+open-source standard, we believe Ibis is the right approach.
+
+Instead of using Snowpark Python for Snowflake, you can use Ibis on Snowflake.
+Instead of using PySpark or pandas on Spark, you can use Ibis on Spark. Instead
+of using the pandas API on BigQuery (built on top of Ibis), you can use Ibis on
+BigQuery. Instead of using PyStarburst on Starburst Galaxy, you can use Ibis on
+Starburst Galaxy. Instead of using the Polars Python on the Polars execution
+engine, you can use Ibis on Polars. Instead of using DataFusion Python on
+DataFusion execution engine, you can use Ibis on DataFusion. Instead of
+executing SQL strings on DuckDB through the Python client, you can use Ibis on
+DuckDB. And so on...
+
+Ibis brings a Python dataframe interface to data platforms that only have SQL,
+and brings a standard Python dataframe interface to data platforms that have
+their own Python dataframe interface. It is the only portable Python dataframe
+that can serve as a standard across the data ecosystem.
+:::
+
+Voltron Data supports Ibis for the same reason it supports Apache Arrow.
+Open-source standards make it easier to build the composable data ecosystem and
+reduce friction for data teams looking to swap out components as their needs
+change. If you're already using Ibis as your frontend, Apache Arrow for data
+interchange, and Substrait for intermediary representation, you can swap in
+Theseus with ease!
```

Comment:
```suggestion
So Voltron Data supports Ibis because it can serve as a universal Python
dataframe API for **any** backend engine, whether you need to query a CSV file
on your laptop, run a big ETL job in the cloud with Snowflake or Starburst, or
process hundreds of terabytes in minutes on an NVIDIA GPU cluster with Theseus.
With Ibis, you have the choice of 20+ backends, and the code you write is the
same regardless of which backend you choose to use.
```

Reply:
Our style guide says to avoid passive voice. Can we change `can serve` to `serves`?

Reply:
nit pick: Can we also remove the "So" at the beginning of the sentence? I don't think it's needed here, and it clutters the sentence.

Reply:
`can serve` is not passive voice.  Passive voice would be if it said `can be served by Ibis`